clang/include/clang/Analysis/Analyses/LifetimePset.h

```diff
@@ -332,12 +341,10 @@ class PSet {
     return !ContainsInvalid && !ContainsNull && !ContainsStatic && Vars.empty();
   }
 
-
   /// Returns true if we look for S and we have S.field in the set.
-  bool containsBase(Variable Var, unsigned Order = 0) const {
-    return llvm::any_of(Vars, [Var, Order](const Variable &Other) {
-      return Var.isBaseEqual(Other) && Order <= Other.getOrder();
-    });
+  bool containsBase(Variable Var) const {
```

Comment:
Rename to `containsParent` for consistency?

Reply:
Done